package.json

```diff
@@ -10,6 +10,7 @@
     "prettier": "^2.1.1",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
+    "react-rating": "^2.0.5",
```

Comment:
can we use https://material-ui.com/components/rating/ .  https://material-ui.com/getting-started/installation/